### PR TITLE
Update sphere_fitting.md

### PR DESCRIPTION
**Pull request by contsili**

### CODE REVIEW

File: example/sphere_fitting.md [hunk 1]
```diff
@@ -96,9 +96,9 @@ This example script demonstrates how to fit a sphere to the MEG Polhemus recorde
     % scale the template MRI
     scale = headshape_sphere.r/scalp_sphere.r;
 
-    T2 = [1 0 0 scalp_sphere.o(1);
-          0 1 0 scalp_sphere.o(2);
-          0 0 1 scalp_sphere.o(3);
+    T2 = [1 0 0 headshape_sphere.o(1);
```
[the remark this answers is not in the document]
Comment (contsili) [reply example/sphere_fitting.md:99]: You are right. I will fix it. I also just read quickly through the code and didn't run it myself.

**Comment by contsili** [comment]
I fixed it

**Comment by schoffelen** [comment]
nice, have you checked?

**Comment by contsili** [comment]
Yes I checked, and it works correctly.

An interesting insight, if scale=1 then T2*S*T1 =  T1*S*T2. In my case (I used my own data) scale was 0.9966 so T2*S*T1 ~  T1*S*T2

**Pull request merged**